File: FC.App/Scripts/Shared/ServiceMessages/GenreFilter.js
var FC;
(function (FC) {
    var Shared;
    (function (Shared) {
        var ServiceMessages;
        (function (ServiceMessages) {
            var GenreFilter = (function () {
                function GenreFilter(filter) {
                    if (filter) {
                        this.GenreID = filter.GenreID;
                        this.ParentID = filter.ParentID;
                        this.Name = filter.Name;
                    }
                }
                return GenreFilter;
            }());
            ServiceMessages.GenreFilter = GenreFilter;
        })(ServiceMessages = Shared.ServiceMessages || (Shared.ServiceMessages = {}));
    })(Shared = FC.Shared || (FC.Shared = {}));
})(FC || (FC = {}));

File: FC.Shared/ServerMessages/GenreFilter.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace FC.Shared.ServerMessages
{
    public class GenreFilter
    {
        public Guid? GenreID { get; set; }
        public Guid? ParentID { get; set; }
        public string Name { get; set; }

    }
}
